@stellar/design-system-website/src/components/ComponentProps/index.tsx

```diff
@@ -57,14 +57,16 @@ export const ComponentProps = ({
   const props = component.children.map((p) => <PropRow p={p} />);
 
   const relatedTypeProps = relatedTypes?.map((t) => {
+    const props = t.children || t?.type?.declaration?.children;
+
     return (
       <Fragment key={`t-${t.id}`}>
         <tr>
           <td colSpan={5}>
             <code>{t.name}</code>
           </td>
         </tr>
-        {t?.type?.declaration?.children?.map((t) => (
```

Comment:
In the case that `t.type.declaration.children` is not present, will this throw an error trying to map over `undefined`?